engine/apps/slack/models/slack_usergroup.py

```diff
@@ -69,7 +69,7 @@ def can_be_updated(self) -> bool:
 
     @property
     def oncall_slack_user_identities(self):
-        users = self.oncall_schedules.get_oncall_users()
+        users = set(self.oncall_schedules.get_oncall_users().values())
```

Comment:
@vadimkerr  I think this won't work since `get_oncall_users` is returning a `Dict[Schedule, List[User]]`:
```python3
>>> schedules = {"foo": ["a", "b"], "bar": ["a", "c"]}
>>> set(schedules.values())
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unhashable type: 'list'
```

we should probably do something like this:
```python3
>>> schedules = {"foo": ["a", "b"], "bar": ["a", "c"]}
>>> set(user for schedule in schedules.values() for user in schedule)
{'a', 'c', 'b'}
```